### PR TITLE
admin panel additions

### LINKED ISSUE #70: As an Admin I want to be able to see the pending blog when a blog is created
Acceptance Criteria: 
Given: on the admin panel
When: a blog is created
Then: it appears in the pending blog field

### LINKED ISSUE #77: As an Admin I want to be able to approve pending blogs
Acceptance Criteria
Given: that the admin is in the admin panel
When: a user has submitted a blog 
Then: the admin can approve the blog 

When: a user has submitted a blog 
Then: the blog doesn't show in the pending blogs field

### PR DESCRIPTION
**Pull request by awqfrxx**
we(alex and I) added:
- when you submit blogs it now goes into pending blogs
- admin can approve these blogs and then they'll go to the blogs section
- added some table headers to make it more clear what's in each column

### CODE REVIEW

**Review by koenhamstra — APPROVED** [review]
Looks good

**Review by Sanyi0001 — APPROVED** [review]

**Review by avanheeren — APPROVED** [review]

File: resources/views/admin.blade.php [hunk 1]
```diff
@@ -54,11 +61,47 @@
...
+                                    <tbody>
+                                    <th>Description</th>
+                                    <th>E-mail</th>
+                                    <th>Edit</th>
+                                    <th>Delete</th>
+                                    <th>Approve</th>
+                                    @forelse($blogs as $blog)
+                                        @if($blog->visibility === 0)
+                                        <tr>
+                                            <td>{{$blog->description}}</td>
```
Comment (Koreyoshi159) [inline resources/views/admin.blade.php]: Space after {{ and before the }}

File: resources/views/welcome.blade.php [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
-    <section class="hero  is-medium  is-bold">
-        <div class="header-img">
-            <img src="./image/sdg-photo.png" />
-        </div>
-    </section>
+{{--    <section class="hero  is-medium  is-bold">--}}
+{{--        <div class="header-img">--}}
+{{--            <img src="./image/sdg-photo.png" />--}}
+{{--        </div>--}}
+{{--    </section>--}}
```
Comment (Koreyoshi159) [inline resources/views/welcome.blade.php:14]: Commented code in a pullrequest?

**Review by Koreyoshi159 — CHANGES_REQUESTED** [review]
see comment

File: resources/views/welcome.blade.php (hunk 2 above)
Comment (awqfrxx) [reply resources/views/welcome.blade.php:14]: not mine, was is main already

**Pull request merged**